<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_Window</class>
 <widget class="QMainWindow" name="Change_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_table">
      <property name="text">
       <string>Выберете таблицу для редакртирования:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget_show_tables"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit_input"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox_table">
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtCurrent</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_canseld">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButton_change">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Измените данные:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
